tcms/testplans/static/testplans/js/get.js

```diff
@@ -103,3 +114,38 @@ function getTestCaseExpandArea(row, testCase) {
         }
     });
 }
+
+function attachEvents() {
+    // update default tester
+    $('.js-test-case-menu-tester').click(function(ev) {
+        // todo this prompt need to be translated
+        var email_or_username = window.prompt('Please type new email or username');
+        if (!email_or_username) {
+            return;
+        }
+        const testCaseId = getCaseIdFromEvent(ev);
```

Comment:
Instead of reading the test case ID from the DOM every time can't you pass it as parameter to `attachEvents` (b/c in the calling context you already have it) and simply use it here ?

Reply:
No, because these events are global, they are not bind to specific tc row. Whenever event is fired I have to check for which row it was triggered.